.github/workflows/build-and-push.yml

```diff
@@ -40,6 +40,19 @@ jobs:
             tag: "fedora"
             quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
+          - dockerfile_path: "18"
```

Comment:
CentOS Stream 8 repository on Quay.io is already prepared https://quay.io/repository/sclorg/nodejs-18-c8s?tab=info

Reply:
thanks.